R/adjust_household_contact_matrix.R

```diff
@@ -0,0 +1,50 @@
+#' @title Adjust Household Contact Matrix
+#' 
+#' This function is used internally within [predict_setting_contacts()]
+#'
+#' @param setting_matrices setting matrix
+#' @param population population information
+#' @param household_size single number of the size of a household
+#' @param household_contact_rate default of 1
+#' @param model_mean_other_householders default of 2.248971
+#'
+#' @return contact matrix that has household adjustment
+#' @author Nick Golding
+#' @export
+adjust_household_contact_matrix <- function(setting_matrices,
+                                            population,
+                                            household_size,
```

Comment:
this should be called `percapita_household_size` so it is clear that the parameter is not the mean household size

Reply:
done